### DISCUSSION TITLE
Having Issue starting CS:GO server

### DISCUSSION THREAD

**Discussion by ErrorDoc404** [pterodactyl/panel#4156] [Support]
i dont know what is issue but here is my Pterodactyl Wings - Diagnostics Report

https://ptero.co/usirytexin.yaml

**Comment by Goksi** [comment]
Are you sure you are using latest version of wings, as this bug should be fixed in [5cd43dd](https://github.com/pterodactyl/wings/commit/5cd43dd4c9dd110aeb6d7cc71067927a0d56506b)
